analyses/snv-callers/lancet-wxs-tests/explore-tcga-pbta.Rmd

```diff
@@ -0,0 +1,591 @@
+    bed_wgs = file.path(data_dir, "intersect_cds_lancet_strelka_mutect_WGS.bed"), # There's no TCGA WGS samples, this is just a place holder and won't be used.
+    bed_wxs = file.path(ref_dir, "intersect_cds_gencode_liftover_WXS.bed")
+  )
+), .id = "datasets")
+
+tmb_cdf_plot(lancet_tmb, plot_title = "Lancet")
+```
+
+[Lancet TMB](plots/tcga-vs-pbta-plots/tmb-cdf-Lancet.png)
+
+Mutect only TMB:
+
+```{r}
+mutect_tmb <- dplyr::bind_rows(list(
+  pbta = calculate_tmb(pbta_mutect,
+    bed_wgs = file.path(data_dir, "intersect_cds_lancet_strelka_mutect_WGS.bed"),
+    bed_wxs = file.path(data_dir, "intersect_cds_lancet_WXS.bed")
+  ),
+  tcga = calculate_tmb(tcga_mutect,
+    bed_wgs = file.path(data_dir, "intersect_cds_lancet_strelka_mutect_WGS.bed"),
+    bed_wxs = file.path(ref_dir, "intersect_cds_gencode_liftover_WXS.bed")
+  )
+), .id = "datasets")
+
+# Make a CDF plot
+tmb_cdf_plot(mutect_tmb, plot_title = "Mutect2")
+```
+
+[Mutect2 TMB](plots/tcga-vs-pbta-plots/tmb-cdf-Mutect2.png)
+
+Strelka only TMB:
+
+```{r}
+strelka_tmb <- dplyr::bind_rows(list(
+  pbta = calculate_tmb(pbta_strelka,
+    bed_wgs = file.path(data_dir, "intersect_cds_lancet_strelka_mutect_WGS.bed"),
+    bed_wxs = file.path(data_dir, "intersect_cds_lancet_WXS.bed")
+  ),
+  tcga = calculate_tmb(tcga_strelka,
+    bed_wgs = file.path(data_dir, "intersect_cds_lancet_strelka_mutect_WGS.bed"),
+    bed_wxs = file.path(ref_dir, "intersect_cds_gencode_liftover_WXS.bed")
+  )
+), .id = "datasets")
+
+# Make a CDF plot
+tmb_cdf_plot(strelka_tmb, plot_title = "Strelka2")
+```
+
+[Strelka2 TMB](plots/tcga-vs-pbta-plots/tmb-cdf-Strelka2.png)
+
+Consensus TMB:
+
+```{r}
+consensus_tmb <- dplyr::bind_rows(list(
+  pbta = calculate_tmb(pbta_consensus,
+    bed_wgs = file.path(data_dir, "intersect_cds_lancet_strelka_mutect_WGS.bed"),
+    bed_wxs = file.path(data_dir, "intersect_cds_lancet_WXS.bed")
+  ),
+  tcga = calculate_tmb(tcga_consensus,
+    bed_wgs = file.path(data_dir, "intersect_cds_lancet_strelka_mutect_WGS.bed"),
+    bed_wxs = file.path(ref_dir, "intersect_cds_gencode_liftover_WXS.bed")
+  )
+), .id = "datasets")
+
+# Make a CDF plot
+tmb_cdf_plot(consensus_tmb, plot_title = "Consensus")
+```
+
+[Consensus TMB](plots/tcga-vs-pbta-plots/tmb-cdf-Consensus.png)
+
+## Overlap of the target regions of both datasets
+
+Download a TCGA Target BED regions file from MC3, and format the chromosome data to 
+be `chr`, save as a TSV file. 
+
+```{r}
+tcga_bed <- readr::read_tsv("https://api.gdc.cancer.gov/data/7f0d3ab9-8bef-4e3b-928a-6090caae885b", col_names = c("chr", "start", "end")) %>%
+  dplyr::filter(!is.na(chr)) %>%
+  dplyr::mutate(chr = paste0("chr", chr)) %>%
+  readr::write_tsv(file.path(ref_dir, "gencode.v19.basic.exome.tsv"),
+    col_names = FALSE
+  )
+```
+
+After using [UCSC BED liftover](https://genome.ucsc.edu/cgi-bin/hgLiftOver) to convert 
+from the Target BED regions for TCGA data from hg19 to hg38 (What the PBTA data is using).
```

Comment:
This will need updating when we produce this file directly (see #597)

Reply:
I think this analysis is getting retired immediately since it was exploratory.